libs/server/Objects/List/ListObjectImpl.cs

```diff
@@ -381,5 +381,101 @@ private void ListPop(byte* input, ref SpanByteAndMemory output, bool fDelAtHead)
                 output.Length = (int)(curr - ptr);
             }
         }
+
+        private void ListSet(byte* input, int length, ref SpanByteAndMemory output)
+        {
+            var isMemory = false;
+            MemoryHandle ptrHandle = default;
+            var _out_ptr = output.SpanByte.ToPointer();
+            var _o_curr = _out_ptr;
+            var _o_end = _o_curr + output.Length;
+
+            ObjectOutputHeader _output = default;
+
+            try
+            {
+                if (list.Count == 0)
+                {
+                    while (!RespWriteUtils.WriteError(CmdStrings.RESP_ERR_GENERIC_NOSUCHKEY.ToArray(), ref _o_curr, _o_end))
```

Comment:
```suggestion
                    while (!RespWriteUtils.WriteError(CmdStrings.RESP_ERR_GENERIC_NOSUCHKEY, ref _o_curr, _o_end))
```

Reply:
I found that VS2022 often gives incorrect prompts at this position. 👻 
I'm not sure if it's caused by branch switching.